### DISCUSSION TITLE
Hide in tab context menu?

### DISCUSSION THREAD

**Discussion by Mike737** [gildas-lormeau/SingleFile#675] [Q&A] [answered]
Can I remove the entry from the tab context menu? I like to use SingleFile only from the toolbar.

**Comment by gildas-lormeau** [comment, marked as answer]
You can set the hidden option `tabMenuEnabled` to `false`, see here for more info https://github.com/gildas-lormeau/SingleFile/wiki/Hidden-options.

**Reply by AveYo** [reply]
> I'm glad to hear you found how to disable this option. It's an hidden option because it's specific to Firefox. I prefer to keep the visible options independent from the browser they're running in. Right now, you're the second user who is not happy with the fact that's it's enabled by default. The feature is implemented for more than one year (maybe 2).  

The elevated annoyance level is definitely because of firefox 88 and upcoming proton menu entries changes ;)
`Reopen Closed Tab` instead of `Undo Close Tab` still gives me pause..
